lib/detail_page.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+
+import 'api/book.dart';
+
+/// Detail Page that displays information on the book
+class DetailPage extends StatelessWidget {
+  /// Creates a Detail Page and ensures book is not null
+  const DetailPage({@required this.book})
+      : assert(book != null, 'book cannot be null');
+
+  /// The book being displayed
+  final Book book;
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+      appBar: AppBar(title: Text(book.title)),
+      body: ListView(
+        children: <Widget>[
+          Image.network(book.bookImage),
+          Container(
+            padding: const EdgeInsets.only(left: 32, right: 32),
+            child: Row(children: <Widget>[
+              Expanded(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    Container(
+                      padding: const EdgeInsets.only(bottom: 8),
+                      child: Text(
+                        book.title,
+                        style: TextStyle(fontWeight: FontWeight.bold),
+                      ),
+                    ),
+                    Text(book.author, style: TextStyle(color: Colors.grey[500]))
```

Comment:
There is not enough contrast here. Take a look at [this](https://contrast-ratio.com/#0xFF9E9E9E-on-%23eee).